src/core/Akka.Remote.Tests.MultiNode/RemoteNodeRestartDeathWatchSpec.cs

```diff
@@ -137,7 +137,7 @@ public RemoteNodeRestartDeathWatchSpecConfig()
             First = Role("first");
             Second = Role("second");
 
-            CommonConfig = DebugConfig(false).WithFallback(ConfigurationFactory.ParseString(
+            CommonConfig = DebugConfig(true).WithFallback(ConfigurationFactory.ParseString(
```

Comment:
I can disable it now - this spec was failing earlier and I needed the additional logs to diagnose why. That's how I discovered the UID issue